src/components/ViewPackagePage/components/package-header.tsx

```diff
@@ -167,6 +210,9 @@ export default function PackageHeader({
                 variant="outline"
                 size="sm"
                 onClick={handleForkClick}
+                className={
+                  !isLoggedIn ? "text-gray-400 hover:text-gray-400" : ""
+                }
```

Comment:
You should pass the `isLoggedIn` value to the `disabled` prop and not the className. The style will be handled for  it in the custom Button

Reply:
You will have to wrap the button inside the `Tooltip` for it!